sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala

```diff
@@ -70,30 +70,6 @@ class ParquetQuerySuite extends QueryTest with ParquetTest with SharedSQLContext
       TableIdentifier("tmp"), ignoreIfNotExists = true, purge = false)
   }
 
-  test("SPARK-15678: not use cache on overwrite") {
```

Comment:
Delete the original test case for following reasons:
1. The cache invalidation is for all file sources
2. The two test cases in ParquetQuerySuite is covered by the new test cases of this PR.
3. The Parquet data source is not migrated to V2 yet.

Reply:
+1